docs/src/main/tut/issue.md

```diff
@@ -138,4 +142,62 @@ The `result` on the right is a [SearchIssuesResult][issue-scala].
 
 See [the API doc](https://developer.github.com/v3/search/#search-issues) for full reference.
 
+## Create a Comment
```

Comment:
Should we have a separate section for comments? like

# Issue API
...
## Issues

### Create an issue
...

## Comments

### Create a comment
...